### PR TITLE
HotFix archiving of changed qualities

### PR DESCRIPTION
**Pull request by labrys**
Fix incorrect grammar
Make backloggedEpisodes more DRY

- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: views/inc_qualityChooser.mako [hunk 1]
```diff
@@ -43,14 +43,24 @@ selected = None
...
+        <h5 id="preferred_text2">But it will stop searching when one of these is downloaded:  <label id="preferred_explanation">${', '.join([Quality.qualityStrings[i] for i in preferred_qualities])}</label></h5>
     </div>
     <div>
         <h5 class="red-text" id="backlogged_episodes"></h5>
     </div>
     <div id="archive" style="display: none;">
-        <h5><b>You can archive episodes with 'downloaded' final status to avoid creating backlog:</b></h5>
+        <h5>
+            <b>
+                Archive all currently downloaded episodes to avoid increasing
```
Comment (fernandog) [inline views/inc_qualityChooser.mako]: it won't archive all

Comment (labrys) [inline views/inc_qualityChooser.mako]: I tried some different wordings and none really worked for me.... I think for now I'll just remove the word all, and leave further clarification for later.

Comment (fernandog) [inline views/inc_qualityChooser.mako]: Archive downloaded episodes that are not currently in backlog (final quality)

**Comment by fernandog** [comment]
it will open backlog Overview in a new tab:

image

**Comment by p0psicles** [comment]
Please do not merge to master and to develop. Create one branch, merge that to master. Then afterwards ff merge master as a merge commit on develop.

**Comment by labrys** [comment]
@p0psicles thats not part of the standard [git-flow](http://nvie.com/posts/a-successful-git-branching-model/) workflow.  If you have a different work-flow or different reference for git-flow please link it.

**Comment by p0psicles** [comment]
Your right that we're not exactly following the git flow regarding releasing and hotfix/bugfixes. But as long as you have a small closed loop,  this way is easier and less error prone, and thus better. The ff-merge is for merging master in dev. You already have your featured merge commit when merge from your hotfix branch to master. 
This way you only have to worry about 1 Pr.

**Pull request closed without merging**